main.js

```diff
@@ -0,0 +1,26 @@
+var products = [
```

Comment:
Can you make your file names all lowercase and dash separated? For example, hoodie2 would be hoodie-2. The Tee.jpg would be tee.jpg.